static/src/javascripts/projects/commercial/modules/consentless/init-safeframes.ts

```diff
@@ -0,0 +1,22 @@
+import { loadScript } from '@guardian/libs';
+
+const initSafeframes = async (): Promise<void> => {
+	const safeframeScripts = [
+		'//cdn.optoutadvertising.com/script/sf/base.min.js',
+		'//cdn.optoutadvertising.com/script/sf/boot.min.js',
+		'//cdn.optoutadvertising.com/script/sf/host.min.js',
+	];
+
+	// These scripts must be loaded in the order set out above
```

Comment:
Strange, they don't need to do that on the demo page? 🤔https://cdn.optoutadvertising.com/script/sf/demo.html

Reply:
As far as I can tell, at least from looking at the un-minified scripts, the host script makes use of existing safeframe logic put in place in some combination of base and boot - the host script errors if not imported in this order.

I'll check this again though and either remove the requirement or amend it. Perhaps in a subsequent PR.